frontend/views/containers/group-settings/InvitationsTable.vue

```diff
@@ -238,21 +245,31 @@ export default ({
     },
     readableExpiryInfo (expiryTime) {
       const timeLeft = expiryTime - Date.now()
-      const MIL = 1000
-      const MIL_MIN = 60 * MIL
-      const MIL_HR = MIL_MIN * 60
-      const MIL_DAY = 24 * MIL_HR
```

Comment:
These constants are already in `time.js` of the project. so replaced these with them.